modules/test/playwright/tests/dynamic-data-mapping-form-web/formSidePanel.spec.ts

```diff
@@ -0,0 +1,113 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {formsPagesTest} from '../../fixtures/formsPagesTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {getRandomInt} from '../../utils/getRandomInt';
+
+export const test = mergeTests(apiHelpersTest, loginTest(), formsPagesTest);
+
+let createdObjectDefinition: ObjectDefinition;
+
+test.afterEach(async ({apiHelpers}) => {
+	if (createdObjectDefinition) {
+		await apiHelpers.objectAdmin.deleteObjectDefinition(
+			createdObjectDefinition.id
+		);
+	}
+});
+
+test.describe('manage form configurations through sidepanel', () => {
```

Comment:
Also, can you explain the relation between this test case and the bug that was fixed by this PR?

Reply:
the select component used as a field on the object entries is the same used on the forms sidebar. 

as i said on the daily meeting, depending on the fix of this bug it could cause the collateral effect on the forms sidebar; it has happened before as you can see on the [LPD-26139](https://liferay.atlassian.net/browse/LPD-26139) comment section.

so i did these tests to prevent both [LPS-203011](https://liferay.atlassian.net/browse/LPS-203011) and [LPD-26139](https://liferay.atlassian.net/browse/LPD-26139) to happen again.

Reply:
also from what i saw, the pattern is to use the uppercase on the description and all lowercase on the test, just like we use in jest. so i will adjust the description to have the upper case; we can discuss with the squad again if you want to change the pattern we are using